docs/developer/HootenannyValidation.asciidoc

```diff
@@ -8,68 +8,67 @@ https://josm.openstreetmap.de/[JOSM] via the
 https://en.wikipedia.org/wiki/Java_Native_Interface[Java Native Interface (JNI)]. This allows 
 Hootenanny C++ core code to invoke JOSM validators by making calls into its Java code. The 
 integration code resides in the `hoot-josm` library and is used by the `validate` command, which 
-uses the `JosmMapValidator` class, and the `JosmMapCleaner` class.
+uses the `JosmMapValidator` class, and the `JosmMapCleaner` class. 
```

Comment:
This document describes running validation standalone outside of the tests and also describes JOSM configuration.